.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Is there a reason why this is needed if there is already CircleCI workflow in place?

Reply:
I've never used circleCI before, I added this out of habit - feel free to remove this if it's redundant.